email_alert_service/models/unpublishing_alert.rb

```diff
@@ -0,0 +1,35 @@
+require "gds_api/email_alert_api"
+
+class UnpublishingAlert
+  def initialize(document, logger, unpublishing_scenario)
+    @document = document
+    @logger = logger
+    @unpublishing_scenario = unpublishing_scenario
+  end
+
+  def trigger
+    logger.info "Received unsubscription notification for #{document['title']}, unpublishing_scenario: #{unpublishing_scenario}, full payload: #{document}"
+    lock_handler.with_lock_unless_done do
+      Services.email_api_client.bulk_unsubscribe(
+        {
+          content_id: document.fetch("content_id"),
+          body: unpublishing_scenario,
+        }.to_json,
+      )
+    rescue GdsApi::HTTPConflict
+      logger.info "email-alert-api returned conflict for #{document['content_id']}, #{document['base_path']}, #{document['public_updated_at']}"
```

Comment:
Do you think that these should be at a higher log level than `INFO` (or even trigger a Sentry alert)?

I think we should try to make it obvious there's a problem and easy to find the logs if there is one given that one of the concerns for the previous version of this feature was a lack of monitoring

Reply:
Perhaps not `GdsApi::HTTPNotFound` as that will come up if a page is unpublished that had no subscribers, but the other two seem to me like they'll only be returned if something has gone wrong in one of our apps